remediation/code-samples/01-countEvens.js

```diff
@@ -22,6 +24,7 @@ function countEvens(array) {
   return array.length;
 }
 
+// PLEASE REVIEW
 function check(output, expected)
 {
   if (output.length !== expected.length){
```

Comment:
Always be mindful of what **type** of data your function is working with.

It looks as though you are treating output and expected as arrays. However, the function `countEvens() `returns an Integer (that we need to compare to an expected Integer count).

We can't take the `length` of an Integer like we can an array. What do we need to do to check if our result is correct? Try just comparing the count we calculated (`output`) to the count we expect (`expected`) using _relational operators._